pkgs/development/python-modules/pyatv/default.nix

```
@@ -11,15 +11,20 @@
 , pytest-asyncio
 , pytestrunner
 , srptools
+, zeroconf
+, fetchFromGitHub
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "pyatv";
-  version = "0.7.2";
+  version = "v0.7.4";
```

v prefix is a version control convention to distinguish between branches and releases, it should be left off
```suggestion
  version = "0.7.4";
```